recipe-client-addon/lib/NormandyDriver.jsm

```diff
@@ -182,11 +182,11 @@ this.NormandyDriver = function(sandboxManager) {
 
     // Study storage API
     studies: {
-      create: sandboxManager.wrapAsync(StudyStorage.create, {cloneArguments: true}),
-      update: sandboxManager.wrapAsync(StudyStorage.update, {cloneArguments: true}),
-      get: sandboxManager.wrapAsync(StudyStorage.get, {cloneInto: true}),
-      getAll: sandboxManager.wrapAsync(StudyStorage.getAll, {cloneInto: true}),
-      has: sandboxManager.wrapAsync(StudyStorage.has),
+      start: sandboxManager.wrapAsync(AddonStudies.start, {cloneArguments: true, cloneInto: true}),
```

Comment:
Ability to install add-ons was one of the things we brought up in the original security review of the sandbox, but it wasn't implemented or concrete at the time, so it may not have gotten enough scrutiny. I'll double-check.

Although, if the answer is no, we have a larger problem, as the entire point of opt-out studies is to install these add-ons. What would be the alternative, shipping the sandbox-level code in the add-on itself and just downloading the argument payload from Shield? Wouldn't that be susceptible to the same issues? Although there's a case to be made that logic errors in the sandbox code are more likely than errors in a static payload.